core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportProfileData.java

```diff
@@ -218,6 +219,40 @@ public void testImportSplitMutationsFile() throws Exception {
         assertEquals("1", clinicalData.get(1).getAttrVal());
     }
 
+    @Test
+    public void testImportSplitFusionsFile() throws Exception {
+        /*
+         * Fusion file split across two files. 
```

Comment:
Give a tiny bit more detail here for context. 

What does "two files" mean? Just add something like, "i.e., somatic fusions and germline fusions"

Otherwise someone looking at this might be confused as to why there would be fusion data stored in multiple files.